internal/tsetse/BUILD.bazel

```diff
@@ -34,3 +34,15 @@ ts_library(
         "//internal/tsetse/rules",
     ],
 )
+
+ts_library(
+    name = "plugin",
+    srcs = [
+        "plugin.ts",
```

Comment:
the naming is confusing, because we already export a symbol called `PLUGIN` from runner.ts.
I think these should be `language_service_plugin` or `ls_plugin` or similar

Reply:
Done